lib/src/serializable.dart

```diff
@@ -138,6 +143,7 @@ class SerializableBus {
   void _deserializeAndCall(
       SerializableModule module, String method, List data) {
     InstanceMirror apiMirror = reflect(module.api);
+    List reflectedData = new List(data.length);
```

Comment:
I'm not sure this would cover all cases. I think the reason my error was specific to `JsonSerializable` was because that was being used in the tests. The only requirement is that the class has a `.fromJson()` constructor. There could be a case where the class implements this constructor without extending/mixing in `JsonSerializable`.

We could add this generic and change the warning message to: "<className> must extend JsonSerializable"